SignalServiceKit/src/Devices/OWSRecordTranscriptJob.m

```diff
@@ -34,14 +36,16 @@ - (instancetype)initWithIncomingSentMessageTranscript:(OWSIncomingSentMessageTra
                                          messageSender:[TextSecureKitEnv sharedEnv].messageSender
```

Comment:
can we remove the message sender dependency now?

Reply:
Great idea.